pkg/config/user_config.go

```diff
@@ -383,6 +384,7 @@ func GetDefaultConfig() *UserConfig {
 				Fetch:                    "f",
 			},
 			Branches: KeybindingBranchesConfig{
+				CopyPullRequestURL:     "c",
```

Comment:
this keybinding conflicts with the 'checkout branch by name' keybinding on line 389. Given it's going to be rarely used (compared to pressing 'o' to create/open the pull request), I would pick a keybinding that requires holding ctrl, though I'm not sure which one specifically would work best. How about "<c-y>" given we use that in the commits panel as an alternative copy-to-clipboard keybinding?

Reply:
I think your "how about ..." is missing the key thing (the actual shortcut) :stuck_out_tongue_closed_eyes: but I'm guessing you wanted to mention `ctrl-y`, am I right?

If that's the case, I completely agree with you, it's better to be consistent

Reply:
haha classic me. Yes, I meant ctrl+y :)

Reply:
I changed the shortcut to be `<c-y>`, but I'm not going to resolve this in case you meant another shortcut

Reply:
So, I'm going to resolve this conversation